hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/cache/FullTableCache.java

```diff
@@ -187,8 +191,8 @@ public void evictCache(List<Long> epochs) {
 
   @VisibleForTesting
   @Override
-  public Set<EpochEntry<CACHEKEY>> getEpochEntrySet() {
-    return epochEntries;
+  public Set<Map.Entry<Long, Set<CACHEKEY>>> getEpochEntrySet() {
```

Comment:
`TestTableCache` unit test checks the size of this set.  Now the result size is equal to epoch count, previously it was cache entry count.  The test is still passing, because it never uses the same epoch in a single test case.

Can you please add a test case for rename (where two entries have the same epoch) in `TestTableCache`?

Reply:
> `TestTableCache` unit test checks the size of this set. Now the result size is equal to epoch count, previously it was cache entry count. The test is still passing, because it never uses the same epoch in a single test case.
> 
> Can you please add a test case for rename (where two entries have the same epoch) in `TestTableCache`?

@adoroszlai  I have added the UT. PTAL.